From: "khvalera@ukr.net" <khvalera@ukr.net>
Date: Tue, 20 Aug 2024 16:46:31 -0700
Subject: [PATCH] input: gen_kheaders

--- a/kernel/gen_kheaders.sh	2025-06-17 21:44:53.000000000 +0300
+++ b/kernel/gen_kheaders.sh	2025-06-17 22:08:23.612873066 +0300
@@ -85,7 +85,7 @@
 # Create archive and try to normalize metadata for reproducibility.
 tar "${KBUILD_BUILD_TIMESTAMP:+--mtime=$KBUILD_BUILD_TIMESTAMP}" \
     --exclude=".__afs*" --exclude=".nfs*" \
-    --owner=0 --group=0 --sort=name --numeric-owner --mode=u=rw,go=r,a+X \
+    --owner=0 --group=0 --numeric-owner --mode=u=rw,go=r,a+X \
     -I $XZ -cf $tarfile -C $cpio_dir/ . > /dev/null
 
 echo $headers_md5 > kernel/kheaders.md5
